tasks/sanity-optional-label-check.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-optional-label-check
+spec:
+  params:
+    - name: sourceImage
```

Comment:
One thing comes to my mind, `sourceImage` here is like `quay.io/org_name/repo_name', but in https://github.com/redhat-appstudio/build-definitions/pull/38/files, it has `registry` and `source-image` and then combine the two params to query in Pyxis. Or use only one param in https://github.com/redhat-appstudio/build-definitions/pull/38/files?
Maybe I am thinking too much.

Reply:
That's a good point. I think that with Pyxis, we're basically filling two URL params, which then makes sense to have as two separate task params. In this one though, `sourceImage` is more free to be specified as someone wishes. I'd personally leave it as it is. What do others think?